[10.0] add new addon to allow to display the category image if the product does not have one (#572)
shopinvader/odoo-shopinvader: shopinvader_category_image_for_product/readme/DESCRIPTION.rst

> This paddon allows to display the category image on a product without images

Reviewer: ```suggestion
This addon allows to display the category image when a product has no images.
```
